hwdb.d/usb.ids

```diff
@@ -15920,6 +15920,7 @@
 	0063  Stream Deck Mini
 	006c  Stream Deck XL
 	006d  Stream Deck original V2
+	009a  Stream Deck Neo
```

Comment:
Please drop the change. This file is automatically picked from upstream, so the change will be dropped.

Reply:
Done.